msf_updates/src/position_pose_msf/position_pose_sensormanager.h

```diff
@@ -186,8 +186,15 @@ class PositionPoseSensorManager : public msf_core::MSF_SensorManagerROS<
     Eigen::Quaterniond yawq(cos(yawinit / 2), 0, 0, sin(yawinit / 2));
     yawq.normalize();
 
-    q = yawq;
-    q_wv = (q * q_ic * q_vc.conjugate()).conjugate();
+    q_wv = yawq.conjugate();
+
+    if (q_vc.w() == 1) {  // If there is no pose measurement, only apply q_wv.
+      q = q_wv;
+    } else {  // If there is a pose measurement, apply q_ic and q_wv to get initial attitud
+      q = (q_ic * q_vc.conjugate() * q_wv).conjugate();
```

Comment:
See how `q_wv` is used during an update:

https://github.com/ethz-asl/ethzasl_msf/blob/master/msf_updates/include/msf_updates/pose_sensor_handler/pose_measurement.h#L313-315

Doesn't this translate to `q_err = (q_wv * q_wi * q_ic).conjugate() * z_q_`? (`q_wi := q`)

https://github.com/ethz-asl/ethzasl_msf/blob/master/msf_updates/include/msf_updates/pose_sensor_handler/pose_measurement.h#L304-309

Here I read `position_err = z_p_ - (C_wv.transpose() * (-p_wv + p_wi + C_q.transpose() * p_ic)) * L`. Note that `C_wv = q_wv.conjugate().toRotationMatrix()`. (`p_wi := p`)

Could it be that the notation is off in the case of `q_wv`?